syntax/ps1.vim

```diff
@@ -82,7 +82,7 @@ syn match ps1Type /\[[a-z_][a-z0-9_.,\[\]]\+\]/
 " Variable references
 syn match ps1ScopeModifier /\(global:\|local:\|private:\|script:\)/ contained
 syn match ps1Variable /\$\w\+\(:\w\+\)\?/ contains=ps1ScopeModifier
-syn match ps1Variable /\${\w\+\(:\w\+\)\?}/ contains=ps1ScopeModifier
+syn match ps1Variable /\${\w\+\(:[[:alnum:]_()]\+\)\?}/ contains=ps1ScopeModifier
```

Comment:
This works for `"${env:ProgramFiles(x86)}\test"` but not `"${ProgramFiles(x86)}"` (tested), both of which work in PowerShell and parse in the command line with the right highlighting. You probably want to make the colon optional.

This works:

```vim
syn match ps1Variable /\${\w\+\(:\?[[:alnum:]_()]\+\)\?}/ contains=ps1ScopeModifier
```

I'll upload a change and get this merged.

Reply:
Thanks for your contribution!